spec/features/posting_and_seeing_peep_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'web_helper'
+
+feature 'User can create a peep' do
+  scenario 'See a message (peep) on chitter' do
+    sign_up(user_name: 'Homer', email: 'homerj@gmail.com', password: 'donuts', password_confirmation: 'donuts')
+    create_peep('Hello world!')
+    expect(page.status_code).to eq 200
+    within 'ul#posts' do
+    expect(page).to have_content('Hello world!')
+    end
+  end
+
+
```

Comment:
I'd separate creating peeps from viewing the list of peeps here.  In order to do this, you can create two peeps directly, without using the UI (`Peep.create(<params>)`), then navigate to the page where the Peeps should be listed and ensure that the list is correct.